src/main/java/com/ot/restaurant/fixtures/CustomerFixture.java

```diff
@@ -0,0 +1,24 @@
+package com.ot.restaurant.fixtures;
```

Comment:
Move this fixture to live under the `test` folder, probably under `test/java.ot/restaurant/fixtures`